yarn.lock

```diff
@@ -11856,7 +11861,7 @@ dateformat@^3.0.0, dateformat@^3.0.3:
   resolved "https://registry.npmjs.org/dateformat/-/dateformat-3.0.3.tgz#a6e37499a4d9a9cf85ef5872044d62901c9889ae"
   integrity sha512-jyCETtSl3VMZMWeRo7iY1FL19ges1t55hMo5yaam4Jrsm5EPL89UQkoQRyiI+Yf4k8r2ZpdngkV8hr1lIdjb3Q==
 
-dayjs@^1.10.4, dayjs@^1.9.4:
+dayjs@^1.10.4:
```

Comment:
Interesting about this one, is I think this is (was?) supposed to be the last PR to use DayJS.  Given that, maybe the remaining dayjs/1.10.4 dep here is actually a dep for another package?  wonder if we should triple check if maybe another new one snuck in or something before closing #4278.

Reply:
I noticed this too, it looks like it's being used by `cypress` (so only for development). Also the date-fns library is also still being pulled down for a number of other dependencies.

Reply:
Yeah; i think your PR is fine, and I did another quick search and sure enough this looks like the only explicit import which you've fixed so we're probably good.   Thanks for your work here!!!!